### ISSUE TITLE
"System.InvalidOperationException: Sequence contains no elements" when creating new project without default engine

### ISSUE THREAD

**Issue by RedGlow** [eumario/godot-manager#69]
**Describe the bug**

If I try to create a new project with the "New Project" button but there is no default engine, Godot Manager crashes with the following stacktrace:

```
Unhandled Exception:
System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <8dc7f7cf628440cbb575eaa126cc9c6d>:0
  at CentralStore.GetVersion (System.String id) [0x00023] in <052e86b1f1884b3796ac5e46249331e3>:0
  at CreateProject.PopulateEngines (System.Boolean updateUse) [0x0000c] in <052e86b1f1884b3796ac5e46249331e3>:0
  at CreateProject.ShowDialog () [0x00035] in <052e86b1f1884b3796ac5e46249331e3>:0
  at ProjectsPanel.OnActionButtons_Clicked (System.Int32 index) [0x0004e] in <052e86b1f1884b3796ac5e46249331e3>:0
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) [0x00000] in <c84d94161bca414384015e66544862cf>:0
  at Godot.GodotSynchronizationContext.ExecutePendingContinuations () [0x0001d] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0
  at Godot.GodotTaskScheduler.Activate () [0x0000c] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0
  at (wrapper native-to-managed) Godot.GodotTaskScheduler.Activate(Godot.GodotTaskScheduler,System.Exception&)

Unhandled Exception:
System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <8dc7f7cf628440cbb575eaa126cc9c6d>:0
  at CentralStore.GetVersion (System.String id) [0x00023] in <052e86b1f1884b3796ac5e46249331e3>:0
  at CreateProject.PopulateEngines (System.Boolean updateUse) [0x0000c] in <052e86b1f1884b3796ac5e46249331e3>:0
  at CreateProject.ShowDialog () [0x00035] in <052e86b1f1884b3796ac5e46249331e3>:0
  at ProjectsPanel.OnActionButtons_Clicked (System.Int32 index) [0x0004e] in <052e86b1f1884b3796ac5e46249331e3>:0
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) [0x00000] in <c84d94161bca414384015e66544862cf>:0
  at Godot.GodotSynchronizationContext.ExecutePendingContinuations () [0x0001d] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0
  at Godot.GodotTaskScheduler.Activate () [0x0000c] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0
  at (wrapper native-to-managed) Godot.GodotTaskScheduler.Activate(Godot.GodotTaskScheduler,System.Exception&)
```

**To Reproduce**
1. Go to the "Godot Engines" page and install at least two engines (e.g.: 4.1.1 and 3.5.2)
2. Set one of them as default (e.g.: 3.5.2)
3. Download / Uninstall the default engine
4. Confirm removal
5. Do NOT set the other engine as default
6. Go to the "Projects" page
7. Click "New Project"

**Expected behavior**
A new project dialog opens.

**Complete the Following:**
 - OS: Windows 11 Home, 10.0.22621 Build 22621
 - Version of Godot Manager: 0.2.2

**Comment by eumario** [comment]
This is a known issue, that will be fixed in 0.3.0.

**Comment by eumario** [comment]
Closing issue, as this was a duplicate of #67.  Commit [0620536](https://github.com/eumario/godot-manager/commit/062053632de3286ea947d515726810fc29fdf354) fixes issue.
